jsexception.access.denied=Sie sind nicht berechtigt, die angeforderte Aktion auszuf\u00fchren. Melden Sie sich als berechtigter Benutzer an, sofern Ihnen dies m\u00f6glich ist.
jsexception.any={0}

jsexception.bean.has.no.method=Die Bean mit dem Namen "{0}" verf\u00fcgt nicht \u00fcber die Methode "{1}".
jsexception.bean.name.not.found.for.interface=F\u00fcr die Schnittstelle "{0}" wurde kein Bean-Name gefunden.
jsexception.bean.no.name=Eine Bean mit dem Namen "{0}" wurde nicht gefunden.
jsexception.bean.not.a.ReportDataSourceService=Die Bean "{0}" ist kein ReportDataSourceService.

jsexception.call.not.supported=Der Aufruf wird von Ressourcen-Lookup-Objekten nicht unterst\u00fctzt.
jsexception.connection.type.not.suported=Der Verbindungstyp wird nicht unterst\u00fctzt.
jsexception.connection.uri.not.specified=Der Verbindungs-URI ist nicht angegeben.
jsexception.could.not.find.resource.with.uri=Die Ressource mit dem URI "{0}" wurde nicht gefunden. 
jsexception.could.not.find.content.resource.with.uri=Die Inhaltsressource mit dem URI "{0}" wurde nicht gefunden.
jsexception.unexpected.resource.type.at.uri=Unerwarteter Typ ({0}) der Ressource mit URI {1}. 

jsexception.default.message=Ein unbekannter Fehler ist aufgetreten. Wenden Sie sich an Ihren Systemadministrator.

jsexception.error.creating.connection=Beim Herstellen der Verbindung ist ein Fehler aufgetreten. Versuchen Sie es erneut.
jsexception.parameter.has.multiple.values=Der Parameter "{0}" weist mehrere Werte auf.

jsexception.parameter.has.no.value=Der Parameter "{0}" weist keinen Wert auf.


jsexception.failed.to.create.connection=Die Verbindung konnte nicht hergestellt werden.
jsexception.failed.to.save.connection=Die Verbindung wurde nicht gespeichert.
jsexception.failed.to.save.connection.changes=Die \u00c4nderungen an der Verbindung wurden nicht gespeichert.
jsexception.file.resource.is.reference=Die Dateiressource "{0}" ist ein Verweis und enth\u00e4lt keine Daten.
jsexception.file.resource.no.match.type=Der deklarierte Typ der Ressource ({0}) entspricht nicht dem tats\u00e4chlichen Typ des Verweises ({1}).
jsexception.folder.already.exists=Es ist bereits ein Ordner mit dem Namen "{0}" vorhanden. W\u00e4hlen Sie einen anderen Namen aus.
jsexception.folder.details.not.found=In den Importinformationen wurden keine Ordnerdetails f\u00fcr den Ordner "{0}" gefunden. 
jsexception.folder.not.found=Der Ordner "{0}" wurde nicht gefunden. 
jsexception.folder.not.found.at=Der Ordner befindet sich nicht in "{0}".
jsexception.folder.duplicate.name=Es befindet sich bereits ein Ordner mit dem Namen "{0}" in "{1}".
jsexception.folder.duplicate.label=Doppelte Ordnerbezeichnung "{0}" im Ordner "{1}".
jsexception.folder.too.long.uri=Der URI f\u00fcr den Ordner ist zu lang. Die Maximall\u00e4nge betr\u00e4gt {0} Zeichen.

jsexception.got.null.schema.source.wrapper=Der Subflow hat einen NULL-Wrapper f\u00fcr die Schemaquelle zur\u00fcckgegeben.
jsexception.got.null.access.grant.source.wrapper=Der Subflow hat einen NULL-Wrapper f\u00fcr die Quelle der Zugriffsgew\u00e4hrung zur\u00fcckgegeben.
jsexception.got.null.connection.back=Der Subflow hat eine NULL-Verbindung zur\u00fcckgegeben.

jsexception.implementation.class.does.not.implement=Die Implementierungsklasse "{0}" implementiert "{1}" nicht.
jsexception.implementation.class.not.found=F\u00fcr "{0}" wurde keine Implementierungsklasse gefunden.
jsexception.import.directory.not.found=Das Importverzeichnis "{0}" wurde nicht gefunden.
jsexception.import.module.not.found=Das Importmodul "{0}" wurde nicht gefunden.
jsexception.import.multiple.values.for.single.parameter=F\u00fcr den Parameter "{0}" des Berichts "{1}", der nur einen Wert akzeptiert, wurden mehrere Parameterwerte erkannt.
jsexception.input.stream.exhausted=Der Eingabe-Stream wurde vor dem Lesen von {0} Bytes beendet.
jsexception.invalid.bean.datasource=Die Bean-Datenquelle "{0}" ist ung\u00fcltig.
jsexception.invalid.class.attribute.for.model.tag=Das Klassenattribut f\u00fcr das Modell-Tag, Ressource "{0}", ist ung\u00fcltig.
jsexception.invalid.jdbc.datasource=Die JDBC-Datenquelle "{0}" ist ung\u00fcltig.
jsexception.invalid.aws.datasource=Die AWS-Datenquelle "{0}" ist ung\u00fcltig.
jsexception.invalid.jndi.jdbc.datasource=Die JNDI/JDBC-Datenquelle "{0}" ist ung\u00fcltig.
jsexception.invalid.olap.datasource=Die OLAP-Datenquelle "{0}" ist ung\u00fcltig.
jsexception.invalid.resource.reference.source=Die Quelle f\u00fcr den Ressourcenverweis "{0}" ist ung\u00fcltig.
jsexception.invalid.session=Die aktuelle Sitzung ist ung\u00fcltig. Vergewissern Sie sich, dass der Server ausgef\u00fchrt wird, und melden Sie sich erneut an.
jsexception.invalid.uri=Der URI "{0}" ist ung\u00fcltig.

jsexception.jar.entry.not.found=Es wurde kein JAR-Eintrag f\u00fcr die URL "{0}" gefunden.
jsexception.jar.malformed.url=Die JAR-URL "{0}" weist eine fehlerhafte Syntax auf.
jsexception.jasperprint.not.found=Das JasperPrint-Objekt "{0}" wurde nicht gefunden.

jsexception.job.has.more.than.one.trigger=Der Auftrag "{0}" weist mehrere Trigger auf.
jsexception.job.no.versions.match=Die Clientversion {0} ist nicht die aktuelle Version ({1}).
jsexception.job.unknown.calendar.trigger.days.type=Der Kalendertrigger "{0}" weist ein unbekanntes Tagesformat auf.
jsexception.job.unknown.interval.unit=Der Berichtsauftrag "{0}" weist eine unbekannte Intervalleinheit auf.
jsexception.job.unknown.start.type=Der Berichtsauftrag "{0}" weist einen unbekannten Starttyp auf.
jsexception.job.unknown.trigger.bean.class=Der Berichtsauftragstrigger "{0}" weist eine unbekannte Bean-Klasse auf.
jsexception.job.unknown.trigger.type=Der Berichtsauftrag "{0}" weist einen unbekannten Triggertyp auf.

jsexception.missing.olap.connection=Die OLAP-Verbindung fehlt.
jsexception.missing.olap.connection.reference=Der Verweis auf die OLAP-Verbindung fehlt.
jsexception.mondrian.jdbc.datasource.has.an.invalid.type=Die JDBC-Datenquelle f\u00fcr die Mondrian-Verbindung weist einen ung\u00fcltigen Typ ({0}) auf.

jsexception.no.connection.selected=Es wurde keine Verbindung ausgew\u00e4hlt.
jsexception.no.export.folder.specified=Es wurde kein Ordner f\u00fcr den Export angegeben.
jsexception.no.export.zip.file.specified=Es wurde keine ZIP-Datei f\u00fcr den Export angegeben.
jsexception.no.import.folder.specified=Es wurde kein Ordner f\u00fcr den Import angegeben.
jsexception.no.import.zip.file.specified=Es wurde keine ZIP-Datei f\u00fcr den Import angegeben.
jsexception.no.input.parameter.specified=Es wurde kein Eingabeparameter angegeben.
jsexception.no.olap.client.connection=Es wurde keine OLAP-Clientverbindung gefunden.
jsexception.no.secure.olap.client.connection=Es wurde keine sichere OLAP-Clientverbindung gefunden.
jsexception.no.olap.model.created.for=Es wurde kein OLAP-Modell f\u00fcr "{0}" erstellt.
jsexception.no.olap.model.name=Es wurde kein OLAP-Modell angegeben.
jsexception.no.olap.model.retrieved=Es konnte kein OLAP-Modell abgerufen werden.
jsexception.no.output.parameter.specified=Es wurde kein Ausgabeparameter angegeben.
jsexception.no.persistent.class.mapped.to="{0}" ist keine persistente Klasse zugeordnet.
jsexception.no.resource.data.provider.found=Es wurde kein Ressourcen-Datenprovider f\u00fcr ID "{0}" gefunden.
jsexception.no.such.object.permission=Die angegebene Objektberechtigung ist nicht vorhanden.
jsexception.no.such.role=Die Rolle "{0}" ist nicht vorhanden.
jsexception.no.such.user=Der Benutzer "{0}" ist nicht vorhanden.
jsexception.no.uri.or.recipient.given=Es wurde kein URI bzw. Empf\u00e4nger angegeben.
jsexception.no.values.to.enumerate=Es wurden keine Werte zum Durchlaufen angegeben.
jsexception.no.values.to.enumerate="{0}" ist kein g\u00fcltiger Ressourcen-URI.
jsexception.null.uri=Null-URI.

jsexception.output.stream.already.closed=Der Ausgabe-Stream ist bereits geschlossen.

jsexception.property.not.a.list=Die Eigenschaft "{0}" von Bean "{1}" ist keine Listeneigenschaft.
jsexception.property.not.a.map=Die Eigenschaft "{0}" von Bean "{1}" ist keine Zuordnungseigenschaft.
jsexception.property.not.a.text=Die Eigenschaft "{0}" von Bean "{1}" ist keine Texteigenschaft.

jsexception.query.datasource.has.an.invalid.type=Die Datenquelle f\u00fcr die Abfrage weist einen ung\u00fcltigen Typ ({0}) auf.

jsexception.report.job.not.found=Es wurde kein Berichtsauftrag mit der ID "{0}" gefunden.
jsexception.report.resource.already.exists.no.overwrite=Die Ausgaberessource "{0}" ist bereits vorhanden, und das Flag zum \u00dcberschreiben von Auftr\u00e4gen ist nicht gesetzt.
jsexception.report.no.content.resource=Die Ressource "{0}" ist keine Inhaltsressource.
jsexception.report.unit.datasource.has.an.invalid.type=Die Datenquelle f\u00fcr die Berichtseinheit weist einen ung\u00fcltigen Typ ({0}) auf.
jsexception.report.unknown.output.format=Das Berichtsausgabeformat "{0}" ist unbekannt.
jsexception.repository.uri.neither.report.nor.folder=Der Repository-URI "{0}" ist weder eine Berichtseinheit noch ein Ordner.
jsexception.resource.already.exists=Die Ressource "{0}" ist bereits vorhanden. W\u00e4hlen Sie einen anderen Namen aus.
jsexception.resource.copyTo.not.implemented="copyTo" ist f\u00fcr die Ressource nicht implementiert.
jsexception.resource.copyFrom.not.implemented="copyFrom" ist f\u00fcr die Ressource nicht implementiert.
jsexception.resource.does.not.exist=Die Ressource "{0}" ist nicht vorhanden.
jsexception.resource.no.match.versions={0}: Die Clientversion {1} entspricht nicht der Repository-Version ({2}).
jsexception.resource.of.type.not.found=Die Ressource "{0}" vom Typ "{1}" wurde nicht gefunden.
jsexception.resource.not.report.unit=Die Ressource "{0}" ist keine Berichtseinheit.
jsexception.resourcelookup.not.a.resource=ResourceLookup ist keine Ressource: {0}.
jsexception.folder.delete.root=Der Stammordner kann nicht gel\u00f6scht werden.
jsexception.report.template.not.found=Der Repository-URI "{0}" ist keine Vorlage.
jsexception.report.template.uri.not.specified=Der Vorlagen-URI ist nicht angegeben.

jsexception.selected.report.not.adhoc=Der ausgew\u00e4hlte Bericht ist kein Ad-hoc-Bericht. Regul\u00e4re Berichtseinheiten werden nicht unterst\u00fctzt.
jsexception.session.timeout.occurred=Bei der OLAP-Sitzung ist ein Timeout aufgetreten.

jsexception.too.many.data.rows=Der Excel-Export ist fehlgeschlagen, weil zu viele Datenzeilen vorliegen. Sie k\u00f6nnen die Exporteinstellungen auf eine Seite pro Arbeitsblatt festlegen.

jsexception.unable.to.complete.request=Das System konnte die Anforderung nicht abschlie\u00dfen. Ihre Unterhaltung ist m\u00f6glicherweise beendet, abgelaufen oder f\u00fcr ung\u00fcltig erkl\u00e4rt worden.
jsexception.undefined.file.type=Der Dateityp wird nicht erkannt.
jsexception.unknown.connection.type=Der Verbindungstyp wird nicht erkannt.
jsexception.unknown.parameter.type.for.multiple.value.input=Der Parametertyp "{0}" l\u00e4sst die Eingabe mehrerer Werte nicht zu.
jsexception.unknown.permission.recipient.type=Der Typ "{0}" des Berechtigungsempf\u00e4ngers wird nicht erkannt.
jsexception.unknown.property.filter.operation=Der Eigenschaftenfilter-Vorgang "{0}" wird nicht erkannt.
jsexception.unknown.timezone=Die Zeitzone "{0}" wird nicht erkannt. 
jsexception.uri.not.found=Der URI "{0}" wurde im Repository nicht gefunden.
jsexception.user.credentials.are.expired=Die Anmeldeinformationen von Benutzer "{0}" sind abgelaufen.
jsexception.user.disabled=Der Benutzer "{0}" ist deaktiviert.
jsexception.user.expired=Der Benutzer "{0}" ist abgelaufen.
jsexception.user.locked=Der Benutzer "{0}" ist gesperrt.
jsexception.user.not.found=Der Benutzer "{0}" wurde nicht gefunden.

jsexception.move.folder.source.uri.ancestor.destination="{0}" kann nicht nach "{1}" verschoben werden. Das Ziel ist ein untergeordnetes Element der Quelle.
jsexception.move.folder.to.same.folder=Der Ordner "{0}" kann nicht in denselben Ordner "{1}" verschoben werden.
jsexception.move.resource.to.same.folder=Die Ressource "{0}" kann nicht in denselben Ordner "{1}" verschoben werden.
jsexception.move.folder.path.already.exists=Der Ordner "{0}" kann nicht nach "{1}" verschoben werden. Der Zielordner enth\u00e4lt bereits ein Objekt mit demselben Namen.
jsexception.move.resource.path.already.exists=Die Ressource "{0}" kann nicht nach "{1}" verschoben werden. Das Ziel enth\u00e4lt bereits ein Objekt mit demselben Namen.
jsexception.copy.resource.referenced.not.readable=Auf die Ressource "{0}" wird von einer anderen Ressource verwiesen, sie kann jedoch nicht gelesen werden.
jsexception.copy.resource.circular.reference=Es wurde ein zirkul\u00e4rer Verweis f\u00fcr die Ressource "{0}" erkannt.
jsexception.folder.moved=Der Ordner "{0}" wurde nach "{1}" verschoben.

report.scheduling.error.filling.report=Der Bericht wurde nicht abgeschlossen. Bei der Ausf\u00fchrung ist ein Fehler aufgetreten.
report.scheduling.error.exporting.report=Der Export nach "{0}" wurde nicht abgeschlossen. Der Vorgang wurde durch einen Fehler unterbrochen.
report.scheduling.error.saving.to.repository="{0}" wurde nicht im Repository gespeichert. Der Vorgang wurde durch einen Fehler unterbrochen.
report.scheduling.error.sending.email.notification=Die Benachrichtigung zum Auftragsfehler wurde nicht abgeschlossen. Beim Senden ist ein Fehler aufgetreten.
report.scheduling.error.system=Der Auftrag wurde nicht abgeschlossen. Bei der Ausf\u00fchrung ist ein Systemfehler aufgetreten.

exception.report.unrecognized.number.type=Nicht erkannter numerischer Typ "{0}"
exception.report.unrecognized.date.type=Nicht erkannter Datums-/Uhrzeittyp "{0}"
exception.report.unsupported.date.input.value=Der Eingabewert "{0}" vom Typ "{1}" kann nicht in einen Datums-/Uhrzeit-Parameter konvertiert werden.
exception.report.unsupported.date.parameter.type=Nicht unterst\u00fctzter Datums-/Uhrzeittyp "{0}"
exception.report.unsupported.input.control.data.type=Der eingegebene Steuerdatentyp "{0}" wird nicht unterst\u00fctzt.

jsexception.view.report.could.not.load=Die Berichtseinheit wurde nicht gefunden, oder sie kann nicht geladen werden.
jsexception.view.page.does.not.exist=Seite {0} wurde im Bericht nicht gefunden.
jsexception.view.report.error=Beim Generieren des Berichts ist ein Fehler aufgetreten.
jsexception.export.unknown.argument=Die Option "{0}" wird nicht erkannt.

jsexception.mondrian.no.connection.for.resource=F\u00fcr die Ressource "{0}" ist keine Mondrian-Verbindung vorhanden.

jsexception.target.org.not.found=Die Zielorganisation "{0}" wurde im Repository nicht gefunden.
jsexception.template.folder.not.found=Der Vorlagenordner wurde im Repository nicht gefunden.

jsexception.search.duplicate.label=Ein Objekt mit der Bezeichnung "{0}" ist im Ordner "{1}" bereits vorhanden.
jsexception.search.duplicate.name=Ein Objekt mit dem Namen "{0}" ist im Ordner "{1}" bereits vorhanden.

report.scheduling.error.saving.data.snapshot.disabled=Das Speichern von Datenschnappsch\u00fcssen wurde deaktiviert.
report.scheduling.error.data.snapshot.not.populated=Der Datenschnappschuss enthielt keine Daten.
report.scheduling.error.data.snapshot.not.persistable=Der Datenschnappschuss kann nicht gespeichert werden.
report.scheduling.error.saving.data.snapshot.failed=Fehler beim Speichern des Datenschnappschusses.

exception.remote.resource.already.exists=De Ressource "{0}" ist bereits vorhanden.
exception.remote.resource.not.found=Die Ressource "{0}" wurde nicht gefunden.
exception.remote.resource.type.not.supported=Der Ressourcentyp "{0}" wird von diesem Vorgang nicht unterst\u00fctzt.
exception.remote.folder.not.found=Der Ordner "{0}" wurde nicht gefunden, oder es handelt sich nicht um einen Ordner.
exception.remote.unexpected.error=Unerwarteter Fehler.
exception.remote.weak.password=Schwaches Kennwort.
exception.remote.resource.not.ready=Die Aufgabe "{0}" ist noch nicht beendet.
exception.remote.no.such.export.process=Es gibt keine Exportaufgabe mit der ID {0}.
exception.remote.no.such.import.process=Es gibt keine Importaufgabe mit der ID {0}.
exception.remote.access.denied=Zugriff verweigert.
exception.remote.modification.not.allowed=Die Ressource "{0}" darf nicht ge\u00e4ndert werden.
exception.remote.illegal.parameter.value.error=Der Wert "{1}" des Parameters "{0}" ist ung\u00fcltig.
exception.remote.mandatory.parameter.error=Der obligatorische Parameter "{0}" fehlt.
exception.remote.import.failed=Der Import ist fehlgeschlagen. Grund: {0}
exception.remote.import.failed.zip.error=Der Import ist fehlgeschlagen. Grund: Die ZIP-Datei kann nicht gelesen werden.
exception.remote.import.failed.content.error=Der Import ist fehlgeschlagen. Grund: Die angegebene ZIP-Datei ist keine g\u00fcltige Exportdatei von JasperReports Server.
exception.remote.resource.querying.not.supported=Die Ressource "{0}" unterst\u00fctzt keine Abfragen.
exception.remote.version.not.match=Die Aktualisierung ist fehlgeschlagen. Die Versionen stimmen nicht \u00fcberein.

#AWS data source specific exception messages
aws.exception.datasource.recovery.base=Bei der Wiederherstellung der AWS-Datenquelle ist eine Ausnahme aufgetreten: {0}.
aws.exception.datasource.recovery.timeout=M\u00f6glicherweise gibt es ein Problem mit der JDBC-URL, den AWS-Sicherheitseinstellungen oder dem Wiederherstellungsvorgang f\u00fcr die AWS-Datenquelle, der bis zu 2 Minuten dauern kann.
aws.exception.datasource.recovery.instance.not.active=Die AWS-Datenquelleninstanz ist nicht aktiv.
aws.exception.datasource.recovery.is.not.possible=Die AWS-Datenquelle kann nicht automatisch wiederhergestellt werden, da Ihr JasperReports Server und die Datenbankinstanz sich nicht in derselben VPC befinden.
aws.exception.datasource.recovery.public.ip.not.provided=Sie m\u00fcssen die '\u00d6ffentliche IP f\u00fcr den Zugriff' dieses Servers in den AWS-Einstellungen angeben, um das automatische Einrichten und Wiederherstellen der AWS-Datenquelle zu erm\u00f6glichen.
aws.exception.datasource.recovery.public.ip.not.determined=Die IP-Adresse der Instanz f\u00fcr die Wiederherstellung der AWS-Datenquelle kann nicht ermittelt werden.
aws.exception.datasource.load.default.credentials=Diese Instanz kann keine <b>EC2-Instanz-Anmeldedaten</b> verwenden, um automatisch eine Verbindung mit Amazon RDS und Amazon Redshift herzustellen, da ihr keine IAM-Rolle (Identity and Access Management, Identit\u00e4ts- und Zugriffsverwaltung) zugewiesen wurde. Bitte lesen Sie <a class='launcher inline' href='http://community.jaspersoft.com/jaspersoft-aws/launch' target='_blank'>Launching Jaspersoft for AWS</a>, um zu erfahren, wie Sie einen Server starten, der automatisch sichere Verbindungen mit AWS-Datenquellen herstellen kann.<br><br>Um dieses Problem zu umgehen, k\u00f6nnen Sie <b>AWS-Anmeldedaten</b> verwenden.
aws.exception.datasource.rds.vpc.recovery=Die VPC-Sicherheitsgruppe "{0}" kann nicht automatisch der RDS-Instanz "{1}" zugewiesen werden. Der Vorgang wird derzeit nicht von der AWS-API unterst\u00fctzt. Weisen Sie die Sicherheitsgruppe manuell Ihrer RDS-Instanz zu.
aws.exception.datasource.accessKey.invalid=Der AWS-Zugriffsschl\u00fcssel ist ung\u00fcltig.
aws.exception.datasource.secretKey.invalid=Der AWS-Geheimschl\u00fcssel ist ung\u00fcltig.

#Cluster specific exception messages
cluster.exception.session.attribute.missing=Auf dem Server ist ein unverwandter Fehler aufgetreten. Ihre Sitzung wurde erneut gestartet. Nicht gespeicherte Arbeit ist jedoch verloren gegangen. Entschuldigen Sie die Unannehmlichkeiten.<p><p>Klicken Sie Sie auf den folgenden Link, um zur Startseite zur\u00fcckzukehren und erneut zu beginnen: <a class="launcher inline" href="home.html">Startseite aufrufen</a>. Falls ein weiterer Fehler auftritt, wenden Sie sich an Ihren Administrator.
cluster.exception.session.attribute.missing.popup=Auf dem Server ist ein unverwandter Fehler aufgetreten. Ihre Sitzung wurde erneut gestartet. Nicht gespeicherte Arbeit ist jedoch verloren gegangen. Entschuldigen Sie die Unannehmlichkeiten.<p><p>Klicken Sie Sie unten auf die Schaltfl\u00e4che, um zur Startseite zur\u00fcckzukehren und erneut zu beginnen. Falls ein weiterer Fehler auftritt, wenden Sie sich an Ihren Administrator.